src/beautifiers/exfmt.coffee

```diff
@@ -0,0 +1,35 @@
+###
+Requires https://github.com/lpil/exfmt
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Exfmt extends Beautifier
+  name: "exfmt"
+  link: "https://github.com/lpil/exfmt"
+  executables: [
+    {
+      name: "exfmt"
+      cmd: "mix"
+      homepage: "https://github.com/lpil/exfmt"
+      installation: "https://github.com/lpil/exfmt"
+      version: {
+        args: ['help','exfmt'],
+        parse: (text) -> text.match(/exfmt-(\d+\.\d+\.\d+)/)[1]
+      }
+      docker: {
+        image: "unibeautify/exfmt"
+      }
+    }
+  ]
+
+  options: {
+    Elixir: false
+  }
+
+  beautify: (text, language, options) ->
+    @run("mix", [
+      'exfmt',
+      @tempFile("input", text, ".ex")
```

Comment:
exfmt has a `--stdin` flag which will allow you to avoid the use of a temp file :)

Reply:
@Glavin001 Glavin, I didn't find any examples using standard input, so leaved temporary file.